components/lock/zookeeper/zookeeper_lock_test.go

```diff
@@ -183,4 +191,7 @@ func TestZookeeperLock_ALock_BLock_AUnlock_BLock_BUnlock(t *testing.T) {
 	})
 	assert.NoError(t, err)
 	assert.Equal(t, lock.SUCCESS, unlock.Status)
+
+	//wait for connection close
+	time.Sleep(time.Second * expireTime)
```

Comment:
It's still unstable if we depend on time. I prefer to mock the goroutine . I submitted a PR to your branch https://github.com/ZLBer/layotto/pull/5
Please review it